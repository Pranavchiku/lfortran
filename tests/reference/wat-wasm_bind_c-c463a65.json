{
    "basename": "wat-wasm_bind_c-c463a65",
    "cmd": "lfortran --no-color --show-wat {infile}",
    "infile": "tests/wasm_bind_c.f90",
    "infile_hash": "ef2aa4a9db67037588bb3e9c1c30b34f3b5667563e65890fc5f06b9a",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "wat-wasm_bind_c-c463a65.stdout",
    "stdout_hash": "a51bb00f24c33ba35771eed240952f83dec95ec4ae01ece13b669a96",
    "stderr": "wat-wasm_bind_c-c463a65.stderr",
    "stderr_hash": "2bf80bc885daa29c2d301ecd575c8f2241854b77fe6af56abdc22bf8",
    "returncode": 0
}