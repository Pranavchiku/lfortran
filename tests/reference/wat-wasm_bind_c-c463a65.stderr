warning: Implied save feature
 --> tests/wasm_bind_c.f90:2:5
  |
2 |     integer  , parameter :: i_max    =  600
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/wasm_bind_c.f90:3:5
  |
3 |     integer  , parameter :: j_max    =  450
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/wasm_bind_c.f90:4:5
  |
4 |     integer :: image_t(j_max, i_max)
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/wasm_bind_c.f90:7:9
  |
7 |         integer, intent(in) :: n, m
  |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/wasm_bind_c.f90:7:9
  |
7 |         integer, intent(in) :: n, m
  |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/wasm_bind_c.f90:8:9
  |
8 |         integer, intent(in) :: A(n,m)
  |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables
